packages/editor/src/components/properties/ModelTransformProperties.tsx

```diff
@@ -140,7 +140,7 @@ export default function ModelTransformProperties({ entity, onChangeModel }: { en
 
       for (const variant of variants) {
         if (clientside) {
-          await clientSideTransformModel(variant as ModelTransformParameters)
+          nuPath = await clientSideTransformModel(variant as ModelTransformParameters)
```

Comment:
what does `nu` mean? can we use full words?

Reply:
haven't you ever heard of nu metal?